code/agents.py

```diff
@@ -67,6 +67,8 @@ def calculate_fish_forces(self):
         for (fish, pos) in neighbours:
 
             magnitude = k / np.linalg.norm(pos - self.position)**power
+            if magnitude > 2:
```

Comment:
These looking like a temporary statement for debugging, should it be removed @fredrikm ?